### ISSUE TITLE
Tabbed Interface - Carousel's controls have accessibility issue 

### ISSUE THREAD

**Issue by olegk-ccohs** [wet-boew/wet-boew#9574]
Hi All: 

We use WET4 carousel widget on our home page (https://www.ccohs.ca/). If you check accessibility of this page using Accessibility Insights for Web, you'll see accessibility errors related to the carousel's controls (arrows and play+pause buttons). If I add role="tab" to each of the controls' <li>s, the error is gone.  How would you suggest updating the widget to add role="tab" then? Is it something you can do at your end? Are there any other fixes of the issue?   

Thanks a lot! 
Oleg

![carousel-controls](https://user-images.githubusercontent.com/85581470/235512233-1ba86ac8-72a7-4075-96d8-2874268c1178.png)
 
[carousel-controls-error.pdf](https://github.com/wet-boew/wet-boew/files/11367519/carousel-controls-error.pdf)

**Comment by olegk-ccohs** [comment]
Hi @kodz1 ! Thank you for commenting. The latest version doesn't have the same issue for carousel buttons but Accessibility Insights for Web checker flags other elements for the same issue. Screenshot is attached.  Thank you.
![updated-wet-carousels-a11y-issue](https://github.com/wet-boew/wet-boew/assets/85581470/f1acff4e-9272-4e29-93c7-50967871881e)
